storage/src/meta/mod.rs

```diff
@@ -447,14 +447,34 @@ impl BlobCompressionContextInfo {
             uncompressed_size: round_up_4k(blob_info.uncompressed_size()),
             chunk_info_array: chunk_infos,
             chunk_digest_array: Default::default(),
+            batch_info_array: Default::default(),
```

Comment:
Could you please help to `impl Default for BlobCompressionContext` so we can simplify the code a bit?